backend/Virkarekisteri/src/Repositories/OrganizationTreeRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Virkarekisteri.Models;
+
+namespace Virkarekisteri.Repositories
+{
```

Comment:
If you switch to a file-scoped namespace as per the earlier comment, it can be done here as well.

Reply:
Fixed